presto-docs/src/main/sphinx/release/release-0.279.rst

```diff
@@ -0,0 +1,64 @@
+* Fix max(x, n) and min(x, n) returning NULL on window aggregations.
+* Fix weighted scheduling for disaggregated coordinator.
+* Add :func:`array_sort_desc` function to sort an array in the descending order.
+* Add :func:`map_remove_null_values` to remove all the entries where the value is null from the given map.
+* Add :func:`map_top_n_values` to return top N values of the provided map. An optional lambda comparator can also be passed as parameter for custom sorting of the values.
+* Add :func:`map_top_n` to truncates map items. Keeps only the top N elements by value.
+* Add :func:`remove_nulls` function to remove null elements from an array.
+* Add ``CONTROL`` as a new ``QueryType``.  The CONTROL queryType represents statements of session control and transaction control types.
+* Add an optimization which removes redundant distinct if the output is already distinct after a group by operation. The optimization is controlled by session property `remove_redundant_distinct_aggregation` which is default to false.
+* Add functions :func:`array_min_by`, :func:`array_max_by`, to find the smallest or largest element of an array when applying a custom measuring function.
+* Add property ``spark_executor_allocation_strategy_enabled`` to auto-tune spark max executor count (``spark.dynamicAllocation.maxExecutors``)  based on input data. Only required if ``spark_resource_allocation_strategy_enabled`` is not already enabled.
+* Add property ``spark_hash_partition_count_allocation_strategy_enabled`` to auto-tune hash partition count (``hash_partition_count``) based on input data. Only required if ``spark_resource_allocation_strategy_enabled`` is not already enabled.
+* Removed two unused session params - `hash_based_distinct_limit_enabled` and `hash_based_distinct_limit_threshold` and the corresponding implementation in favor of the `quick_distinct_limit_enabled` feature.
+* Extend :func:`array_frequency`, :func:`array_duplicates`, :func:`array_has_duplicates`, :func:`array_intersect(array(array(E))` to accept any type as input instead of only varchar/double.
+* Introduced a new plugin type for tracing. The legacy tracer module can be replaced by the new plugin for loading customized tracing infrastructure.
+
+Delta Lake Connector Changes
+____________________________
+* Upgrade Delta Standalone to 0.6.0.
+
+Elasticsearch Connector Changes
+_______________________________
+* Add support for Elasticsearch user and password authentication. :issue:`15909`.
+
+SPI Changes
+___________
+* Rename ``ConnectorMaterializedViewDefinition`` to  ``MaterializedViewDefinition``.
+
+Hive Changes
+____________
+* Fix a bug when ``optimize_metadata_queries`` was set to true where queries with aggregations on partition columns and filters on row subfields could return wrong results.
+* Disable S3 Select pushdown when the query does not have a predicate or projection.
+
+Iceberg Changes
+_______________
+* Update Iceberg from 0.14.1 to 1.0.0.
```

Comment:
Should be 
```
* Update Iceberg from 0.14.1 to 1.1.0.
```
because [Upgrade Iceberg from 1.0.0 to 1.1.0 #18709](https://github.com/prestodb/presto/pull/18709/files) also included in the release